### ISSUE TITLE
Keywords in column names are not supported due to missing escaping support

### ISSUE THREAD

**Issue by Christoph-W** [JDBDT/jdbdt#67]
Given a database system view with column names being keywords can not be handled by JDBDT.
It is possible to create table tables meta data:
```java
partitionsTable = JDBDT.table("USER_TAB_PARTITIONS")
    .columns("TABLE_NAME", "PARTITION_NAME", "HIGH_VALUE", "\"INTERVAL\"")
    .build(database)
```
But using it during data preparation
```java
JDBDT.builder(partitionsTable)
    .sequence(...)
    .nullValue("\"INTERVAL\"")
```
fails with `org.jdbdt.InvalidOperationException: Invalid column name: '"INTERVAL"'.`
Using `.remainingColumnsNull()` does not reuse the metadata correctly ending in not escaped column name usage with `Syntax error in SQL statement`.
Also alternative escaping with \` fails. Even enabling of `Option.CASE_SENSITIVE_COLUMN_NAMES` did not help.

**Comment by edrdo** [comment]
@Christoph-W This is a tricky edge case where the behavior should be highly dependent on the database engine at stake; which one are you using? Have you tried using `nullValue("INTERVAL")` (I guess it should not work) ? Try also: 1) defining the column name in the definition of `partitionsTable` as `"\"interval\""` rather than `"\"INTERVAL\""` with  `Option.CASE_SENSITIVE_COLUMN_NAMES` enabled and 2) later use `"interval"` in data set builder operations.

**Comment by Christoph-W** [comment]
The following combination seems to be the single valid solution:
```java
database.enable(Option.CASE_SENSITIVE_COLUMN_NAMES);
partitionsTable = JDBDT.table("USER_TAB_PARTITIONS")
    .columns("TABLE_NAME", "PARTITION_NAME", "HIGH_VALUE", "\"interval\"")
    .build(database);
...
JDBDT.builder(partitionsTable)
    .sequence(...)
    .nullValue("\"interval\"")
```
It does not seem reasonable to me to discard the support of upper case keywords in column and/or table names.
Finally, I would be happy if I could find hints in the documentation that keywords can only be escaped if they are written in lower case.

**Comment by edrdo** [comment]
> It does not seem reasonable to me to discard the support of upper case keywords in column and/or table names.
> I would be happy if I could find hints in the documentation that keywords can only be escaped if they are written in lower case.

@Christoph-W Yes that's how it works and the documentation is not explicit. The overall scheme for escaped column names is currently fragile.

**Comment by edrdo** [comment]
@Christoph-W I have been examining this issue, trying to improve support for the upcoming 1.4.0 release but a small incremental fix at this time is not possible. The overall scheme will have to be re-examined in a future release.
